src/App.js

```diff
@@ -1,10 +1,158 @@
-import React from 'react'
-import { AppContainer } from './components/AppContainer'
+import React from "react";
+import NavBar from "./Components/NavBar";
+import axios from "axios";
+import "./styles/styles.css";
+import "./styles/home.css";
+import banner from "./assets/banner.png";
+import ShoppingCart from "./Components/ShoppingCart";
 
-function App() {
-	return (
-        <AppContainer />
-	)
+import Sell from "./Components/Sell";
+import UserRegister from "./Components/UserRegister";
+import PageProduct from "./Components/PageProduct";
+const baseURL =
+  "https://us-central1-labenu-apis.cloudfunctions.net/fourUsedOne/products";
+
+class App extends React.Component {
+  state = {
+    products: [],
+    home: true,
+    pageSell: false,
+    pageRegister: false,
+    pageCart: false,
+    pageProd: false,
+    cartProds: [],
+    pageProductInfo: "",
+
+  };
+
+  componentDidMount() {
+    this.renderProducts();
+  }
+
+  renderProducts = () => {
+    axios
+      .get(baseURL)
+      .then((response) => this.setState({ products: response.data.products }));
+  };
+
+  // onClickBuy = (id) => {
+  //   let prodCart = [...this.state.cartProds];
+  //   this.state.products.map((prod) => prod.id === id && prodCart.push(prod));
+  //   this.setState({ pageProd: true, cartProds: prodCart });
+  // };
+
+
+
+  onClickHome = () => {
+    this.setState({
+      home: true,
+      pageProd: false,
+      pageCart: false,
+      pageSell: false,
+      pageRegister: false,
+    });
+  };
```

Comment:
Usando so uma propriedade no estado, como falei ali em cima, todas essas linhas poderiam ser substituídas por uma só this.setState({ currentPage: "Home" })